docs/gravitino-server-config.md

```diff
@@ -88,6 +88,50 @@ Gravitino server uses tree lock to ensure the consistency of the data. The tree
 
 Refer to [Iceberg REST catalog service](iceberg-rest-service.md) for configuration details.
 
+### Event listener configuration
+
+Gravitino provides event listener mechanism to allow users to capture the event provides by Gravitino server to integrate some custom operations.
+
+To leverage the event listener, you must implement the `EventListenerPlugin` interface and place the resulting JAR file in the classpath of the Gravitino server. Then, add configurations to gravitino.conf to enable the event listener.
+
+| Property name                              | Description                                                                                            | Default value | Required | Since Version |
+|--------------------------------------------|--------------------------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `gravitino.eventListener.names`            | The name of the event listener, For multiple listeners, separate names with a comma, like "audit,sync" | (none)        | Yes      | 0.5.0         |
+| `gravitino.eventListener.{name}.className` | The class name of the event listener, replace `{name}` with the actual listener name.                  | (none)        | Yes      | 0.5.0         | 
+| `gravitino.eventListener.{name}.{key}`     | Custom properties that will be passed to the event listener plugin.                                    | (none)        | Yes      | 0.5.0         | 
+
+#### Event
+
+Gravitino generates an event following the completion of CREATE, DROP, PURGE, ALTER, LOAD, LIST operations. The events can be hooked to the following resources:
+
+- table
+- fileset
+- topic
+- schema
+- catalog
+- metalake
```

Comment:
You'd better make a table to list all the events for each category.

Reply:
done